source/_docs/guides/composer_convert.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Converting a Standard Drupal 8 Site to a Composer Managed Site
+description: Drupal 8 sites often require the usage of Composer to manage site dependencies. The need to begin using Composer for a site build can often surface after a site is in development, necessitating a divergance from the Pantheon managed upstream.
+draft: true
+contributors: dustinleblanc
+---
+<div class="alert alert-info">
+  <h4 class="info">Note</h4><p markdown="1">Converting to a Composer managed site *removes* the ability to [apply updates via the site dashboard](/docs/upstream-updates/). This is for advanced users who are comfortable taking complete responsibility for the management of site updates. 
+</p>
+</div>
+
+## Before You Begin
+
+ - Make sure to review our documentation on [Git](/docs/git) and have `git` installed.
+ - Make sure to review our documentation on [Composer](/docs/composer) and have `composer` installed.
+ - Make sure to clone your _current_ Pantheon site repository to a working directory on your workstation.
+
+## Checkout a New Branch
+We're about to make some massive changes to the codebase, it is wise to do this work on a branch as it might take you sometime to complete and rolling back can be complicated:
+```sh
+git checkout -b composify
+```
+
+## Create a New Project with Composer
+
+On your local workstation, from the site directory of your Pantheon site, execute the following command:
+
+```command
+cd .. && composer create-project pantheon-systems/example-drops-8-composer --stability=alpha my-site-composer
+```
+Make sure to substitute your project's name for `my-site`
+
+This will create a new directory based on the example project [pantheon-systems/example-drops-8-composer](https://github.com/pantheon-system/example-drops-8-composer) in the `my-site-composer` directory. The directory structure should now look like this;
+```sh
+Sites
+|-my-site
+|-my-site-composer
+```
+
+## Add in your Contrib/Custom code
+
+What makes your site code unique is your selection of contributed modules and themes and any custom modules or themes your development team has created. These customizations need to be replicated in our new project structure.
+
+### Contributed Code
+#### Modules
+A Composer managed site should be able to include all custom code via Composer. Begin by reviewing your existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`. Add these modules to your new codebase using Composer by doing the following for each module in the `my-site-composer` directory:
+```sh
+composer require drupal/module_name
+``` 
+Where `module_name` is the machine name of the module in question
+
+#### Themes
+Repeat the same process with themes, checking `/themes`, `/themes/contrib`, `/sites/all/themes`, and `/sites/all/themes/contrib`. Use Composer in the same way as above to require these.
+
```

Comment:
Do we want to consider advising `https://www.drupal.org/project/composer_generate` or similar? Someone made a similar but better-maintained version of this, but I don't remember where it is, and drupal.org is 5xx-erroring right now so I can't search for it.

Reply:
`composer_generate` is a Drush extension; it should be installed to somewhere like `~/.drush`

Reply:
punted on this to GSD